server/utils/applications/withdrawalReasons.ts

```diff
@@ -35,9 +34,16 @@ export const placementNoLongerNeededOptions = filterByType<OtherReasons>(
   withdrawalReasons,
 )
 
-export const withdrawalRadioOptions = [
+export const withdrawalRadioOptions = (conditionalValue: string) => [
   { divider: 'The application is no longer needed' },
   ...convertKeyValuePairToRadioItems(placementNoLongerNeededOptions, undefined),
+  {
+    text: 'The placement is not needed for another reason',
+    value: 'other',
+    conditional: {
+      html: conditionalValue,
+    },
+  },
```

Comment:
I find the variable names in this file a little confusing now they contain just two of the three withdrawal reasons

You could probably add an optional parameter to `convertKeyValuePairToRadioItems` along the lines of `conditionals?: Record<keyof T, RadioItem['conditional']>` and add it to the relevant item there if present?

Alternatively a quicker solution would be to add `WithoutOther` to the two variable names (`placementNoLongerNeededReasonsWithoutOther`, `placementNoLongerNeededOptionsWithoutOther`, and `withdrawalReasons` if the change there isn't reverted) here might help?